specification/redis/resource-manager/Microsoft.Cache/stable/2023-08-01/redis.json

```diff
@@ -744,6 +744,80 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/redis/{name}/flush": {
+      "post": {
+        "tags": [
+          "Redis"
+        ],
+        "operationId": "Redis_FlushCache",
+        "x-ms-examples": {
+          "RedisCacheFlush": {
+            "$ref": "./examples/RedisCacheFlush.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location",
+          "final-state-schema": "../../../../../common-types/resource-management/v5/types.json#/definitions/OperationStatusResult"
+        },
+        "description": "Deletes all of the keys in a cache.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Redis cache."
+          },
```

Comment:
We can use the cacheName parameter here instead https://github.com/Azure/azure-rest-api-specs/blob/cc9ae458d11ac88cb79885debbf1d3f7239c4512/specification/redis/resource-manager/Microsoft.Cache/stable/2023-08-01/redis.json#L3343